Resources/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "bazinga-translator",
+  "version": "1.0.0",
```

Comment:
The version used should be the same as the BazingaJsBundle. That way we keep both on track, if in the future we want to update the javascript. Like reactjs does in all of their internal components.